wherehows-web/app/components/datasets/containers/dataset-health.ts

```diff
@@ -163,4 +179,11 @@ export default class DatasetHealthContainer extends Component {
       currentSeverityFilter: filterType === 'severity' && newFilterName !== currentSeverityFilter ? newFilterName : ''
     });
   }
+
+  constructor() {
+    super(...arguments);
+    console.log('constructing');
+    console.log(this.tabSelected);
+    set(this, 'tabSelected', this.tabSelected);
```

Comment:
`set(this, 'tabSelected', this.tabSelected)`? what's the intention here, isn't this a no-op, seems like the constructor does not need to be overridden  at all

Reply:
Constructor should be removed. It was doing more in a previous iteration and I was trimming it down.